packages/env-options/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@endo/env-options",
+  "version": "0.1.0",
+  "private": null,
+  "description": "Description forthcoming.",
```

Comment:
I wonder if we should have the package generation script detect an interactive terminal to prompt for this kind of data.

Reply:
@kriskowal , @gibson042 is also making a suggestion here for the script.

In any case, fixed here, so resolving.